src/app/component/ubs/components/ubs-order-details/ubs-order-details.component.scss

```diff
@@ -0,0 +1,297 @@
+.container {
+  box-shadow: 1px 4px 8px rgb(100 114 125 / 18%);
+  padding: 10px 25px 40px;
+  color: var(--ubs-primary-grey);
+}
+
+h3 {
+  font-weight: bold;
+}
+
+.header {
+  width: 100%;
+  margin: 3rem 0 0;
+
+  h3 {
+    margin-bottom: 1rem;
+  }
+
+  p {
+    color: var(--ubs-secondary-grey);
+    margin-bottom: 3rem;
+  }
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 1rem 0 1.2rem 0;
+  border-top: 1px solid var(--ubs-primary-light-grey);
+  border-bottom: 1px solid var(--ubs-primary-light-grey);
+}
+
+label {
+  color: var(--ubs-secondary-grey);
+  padding-left: 0;
+}
+
+label:nth-child(2),
+label:nth-child(3) {
+  margin-left: 1.1rem;
+}
+
+label:nth-child(4),
+label:nth-child(5) {
+  padding-left: 1.1rem;
+}
+
+label:nth-child(5) {
+  padding-left: 4rem;
+}
+
+.form-group {
+  padding: 0;
+}
+
+.form-control {
+  height: 3rem;
+  background: none;
+  padding-left: 0;
+  border: none;
+}
+
+.count input {
+  border: 1px solid var(--ubs-secondary-grey);
+  padding-left: 6px;
+}
+
+.custom-control-label {
+  padding-top: 3px;
+}
+
+.custom-control-label::before {
+  background-color: var(--ubs-primary-white);
+}
+
+.custom-radio .custom-control-input:checked ~ .custom-control-label::before,
+.custom-radio .custom-control-input:checked ~ .custom-control-label::after {
+  background-color: var(--ubs-primary-green);
+  border-radius: 50%;
+}
+
+.sum {
+  width: 6rem;
+  margin-left: 3rem;
+
+  .form-control {
+    padding-left: 0;
+    text-align: right;
+    font-weight: 700;
+  }
+}
+
+button {
+  width: 9rem;
+  height: 3rem;
+  border-radius: 0;
+  border: none;
+}
+
+.primary-global-button {
+  background: var(--ubs-primary-green);
+}
+
+.middle {
+  width: 100%;
+  margin-top: 10px;
+  padding: 30px 0;
+  justify-content: space-between;
+  border-bottom: 1px solid var(--ubs-primary-light-grey);
+}
+
+.totalInfo {
+  text-align: right;
+  width: 25%;
+
+  p {
+    display: flex;
+    justify-content: space-between;
+    color: var(--ubs-secondary-grey);
+    padding-right: 0.5rem;
+  }
+
+  span {
```

Comment:
Why do you need span as a block element?

Reply:
fixed